assets/xml/objects/object_ane.xml

```diff
@@ -0,0 +1,59 @@
+<Root>
+    <File Name="object_ane" Segment="6">
+        <!-- Cucco Lady Skeleton -->
+        <Skeleton Name="gCuccoLadySkel" Type="Flex" LimbType="Standard" Offset="0xF0"/>
+
+        <!-- Cucco Lady Skeleton Limbs -->
+        <Limb Name="gCuccoLadyRootLimb" Type="Standard" Offset="0x000"/>
+        <Limb Name="gCuccoLadyLimb_00C" Type="Standard" Offset="0x00C"/>
+        <Limb Name="gCuccoLadyLimb_018" Type="Standard" Offset="0x018"/>
+        <Limb Name="gCuccoLadyLimb_024" Type="Standard" Offset="0x024"/>
+        <Limb Name="gCuccoLadyLimb_030" Type="Standard" Offset="0x030"/>
+        <Limb Name="gCuccoLadyLimb_03C" Type="Standard" Offset="0x03C"/>
+        <Limb Name="gCuccoLadyLimb_048" Type="Standard" Offset="0x048"/>
+        <Limb Name="gCuccoLadyTorsoLimb" Type="Standard" Offset="0x054"/>
+        <Limb Name="gCuccoLadyLimb_060" Type="Standard" Offset="0x060"/>
+        <Limb Name="gCuccoLadyLimb_06C" Type="Standard" Offset="0x06C"/>
+        <Limb Name="gCuccoLadyLimb_078" Type="Standard" Offset="0x078"/>
+        <Limb Name="gCuccoLadyLimb_084" Type="Standard" Offset="0x084"/>
+        <Limb Name="gCuccoLadyLimb_090" Type="Standard" Offset="0x090"/>
+        <Limb Name="gCuccoLadyLimb_09C" Type="Standard" Offset="0x09C"/>
+        <Limb Name="gCuccoLadyHeadLimb" Type="Standard" Offset="0x0A8"/>
+
+        <!-- Cucco Lady Limb DisplayLists -->
+        <DList Name="gCuccoLadyDressDL" Offset="0x6DF0"/>
+        <DList Name="gCuccoLadyDL_6D20" Offset="0x6D20"/>
+        <DList Name="gCuccoLadyDL_6C38" Offset="0x6C38"/>
+        <DList Name="gCuccoLadyShoe1DL" Offset="0x69B0"/>
+        <DList Name="gCuccoLadyDL_68E0" Offset="0x68E0"/>
+        <DList Name="gCuccoLadyDL_67F8" Offset="0x67F8"/>
+        <DList Name="gCuccoLadyShoe2DL" Offset="0x6570"/>
+        <DList Name="gCuccoLadyTorsoDL" Offset="0x6280"/>
+        <DList Name="gCuccoLadyDL_6148" Offset="0x6148"/>
+        <DList Name="gCuccoLadyDL_6018" Offset="0x6018"/>
+        <DList Name="gCuccoLadyDL_5E28" Offset="0x5E28"/>
+        <DList Name="gCuccoLadyDL_5CF0" Offset="0x5CF0"/>
+        <DList Name="gCuccoLadyDL_5BC0" Offset="0x5BC0"/>
+        <DList Name="gCuccoLadyDL_59D0" Offset="0x59D0"/>
+        <DList Name="gCuccoLadyHeadDL" Offset="0x54D8"/>
+
+        <!-- Palettes -->
+        <Texture Name="gCuccoLadyTLUT" OutName="tlut" Format="rgb5a1" Width="16" Height="16" Offset="0x0108"/>
+
+        <!-- Textures -->
+        <Texture Name="gCuccoLadyHairTex" OutName="hair" Format="ci8" Width="8" Height="16" Offset="0x0308"/>
```

Comment:
all ci formats should set TlutOffset like
```suggestion
        <Texture Name="gCuccoLadyHairTex" OutName="hair" Format="ci8" Width="8" Height="16" Offset="0x0308" TlutOffset="0x0108"/>
```
same with other files

Reply:
See https://github.com/shawlucas/oot/pull/4

Reply:
object_tk was giving me a lot of problems and I decided to not do it. i forgot to remove it from trello